eodatasets3/prepare/landsat_l1_prepare.py

```diff
@@ -452,6 +519,9 @@ def _normalise_dataset_path(input_path: Path) -> Path:
         input_path = input_path.parent
 
     mtl_files = list(input_path.rglob("*_MTL*"))
+
+    if len(mtl_files) > 1:
+        mtl_files = list(input_path.rglob("*_MTL.txt"))
```

Comment:
It took me a moment to understand this addition. I think we should just add `.txt` to the original statement. It only expects a text file from it, and will fall over otherwise

Reply:
Excellent, done.